changelog/6341.bugfix.rst

```diff
@@ -0,0 +1 @@
+Disallow passing session/config to Node.from_parent.
```

Comment:
Not sure this changelog is needed, given that `Node.from_parent` was never released.